setup.cfg

```diff
@@ -63,4 +63,20 @@ ignore =
 
 [mypy]
 files = pybind11
-strict = True
+python_version = 2.7
+warn_unused_configs = True
+
+# Currently (0.800) identical to --strict
```

Comment:
Is this something to urgently add now?

Reply:
The old version was broken, there isn't a `strict=True` flag. This is just writing out exactly what `--strict` is on the command line (it passes).

Reply:
But well, if it's the same as before, it can't really hurt either, I guess. Still, it's a bit of an odd change in just a changelog + version update.

Reply:
In the release procedure "check the setup.cfg" is listed, and I noticed this when I was checking the setup.cfg.

Reply:
Since mypy gets checked by CI, and these settings do not affect downstream projects at all, I think it's fine.

Reply:
Aaaah, I see, OK. Shouldn't our builds have been broken, then? :-)

Reply:
We didn't have `warn_unused_configs = True`, so it just ignored the fact we had an invalid config line here.